src/main/java/uk/gov/cabinetoffice/csl/controller/CslServiceController.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.cabinetoffice.csl.controller;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import static org.springframework.http.HttpStatus.OK;
+
+@Slf4j
+@RestController
+@RequestMapping(path = "/csl")
+public class CslServiceController {
+
+    @GetMapping(path = "/test/{input}", produces = "application/json")
```

Comment:
Is this endpoint needed? Seems like this is a temporary endpoint.

Reply:
It is a temporary endpoint for the initial security setup testing which will be removed after integrating with the  identity service as part of the LC-1591.